build/DotNetSdkVersions.txt

```diff
@@ -1,3 +1,3 @@
 # Each line represents arguments for the .NET SDK installer script (https://learn.microsoft.com/dotnet/core/tools/dotnet-install-script)
--Channel 8.0.3xx
 -Channel 3.1 -Runtime dotnet
+-Channel 8.0.3xx
```

Comment:
installing 3.1 before installing 8.0 fixes an issue on arm64 macOS because otherwise we'd get the `dotnet` host from 3.1 and that one didn't support arm64.